src/main/java/net/pms/database/MediaTableFiles.java

```diff
@@ -1184,4 +1097,31 @@ public static DLNAThumbnail getThumbnailByTitle(final Connection connection, fin
 
 		return null;
 	}
+
+	//TODO : review this
```

Comment:
Does this still need to be done?

Reply:
yes, it will need a complete review on it's usage on a next PR.